yarn.lock

```diff
@@ -5,34 +5,80 @@
 ansi-styles@^3.2.1:
   version "3.2.1"
   resolved "https://registry.yarnpkg.com/ansi-styles/-/ansi-styles-3.2.1.tgz#41fbb20243e50b12be0f04b8dedbf07520ce841d"
+  integrity sha512-VT0ZI6kZRdTh8YyJw3SMbYm/u+NqfsAxEpWO0Pf9sq8/e94WxxOpPKx9FR1FlyCtOVDNOQ+8ntlqFxiRc+r5qA==
   dependencies:
     color-convert "^1.9.0"
 
+array-differ@^2.0.3:
```

Comment:
I'm curious where these changes are coming from since you didn't modify any of the direct dependencies of this package. The integrity lines make sense but I'm confused why there are all these additional packages.

Reply:
Yeah, I just ran `yarn update` and that's what happened. But sure, I'll revert

Reply:
So i reverted the changes to `yarn.lock`, but when I then cleaned and ran `yarn` anew, it removed the `typescript-eslint-parser@^15.0.0` reference in there… What do you think, should we let yarn have its way?

Reply:
I think this is fine, thanks!